oz-docs.js

```diff
@@ -149,11 +149,9 @@ async function startServer(port) {
       verbose,
       root,
       reload: true,
+      port,
     });
-    server.listen(port);
     chokidar.watch(root).on('all', () => server.reload());
-
-    process.on('exit', () => server.close());
```

Comment:
Isn't this leaving the port open if the server forcefully closes?

Reply:
https://github.com/frangio/servbot/blob/ff5a374d22bc2c7e102f4f95efc0632538b21beb/index.js#L42-L45

Processes are closed by servbot now. See the diff:
https://github.com/frangio/servbot/compare/v0.2.5...v0.3.0-1